manifests/minio-operator.v4.1.2.clusterserviceversion.yaml

```diff
@@ -8,37 +8,66 @@ metadata:
             "apiVersion": "minio.min.io/v2",
             "kind": "Tenant",
             "metadata": {
-              "name": "minio"
+              "name": "minio",
+              "labels": {
+                "app": "minio"
+              },
+              "annotations": {
+                "prometheus.io/path": "/minio/v2/metrics/cluster",
+                "prometheus.io/port": "9000",
+                "prometheus.io/scrape": "true"
+              }
             },
             "spec": {
-              "pools": {
-                "servers": 4,
-                "volumesPerServer": 4,
-                "volumeClaimTemplate": {
-                  "metadata": {
-                    "name": "data"
-                  },
-                  "spec": {
-                    "accessModes": [
-                      "ReadWriteOnce"
-                    ],
-                    "storageClassName": "local-storage",
-                    "resources": {
-                      "requests": {
-                        "storage": "1Ti"
+              "image": "minio/minio:RELEASE.2021-06-17T00-10-46Z",
+              "imagePullPolicy": "IfNotPresent",
+              "credsSecret": {
+                "name": "minio-creds-secret"
+              },
+              "pools": [
+                {
+                  "servers": 4,
+                  "name": "pool-0",
+                  "volumesPerServer": 4,
+                  "volumeClaimTemplate": {
+                    "metadata": {
+                      "name": "data"
```

Comment:
Minor not, but if this has no other effect beyond visibility, can we prefix with "minio-" so the resulting PVC are more easily associated with MinIO?

Disregard if this field needs to be data.

Reply:
makes sense, renamed to `minio-`.